docker/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.1'
+
+services:
+  solr:
+    image: solr4
+    restart: always
+    ports:
+      - 8983:8983
```

Comment:
the best practice is to take ports into quotes (like `- "8983:8983"`), otherwise YAML may sometimes interpret them as a single sexagesimal number. (not this one though)

Reply:
Hah yeah I remember reading about this, I think in a Puppet context? Good catch, fixed the two `ports` assignments in the file now.